docs/source/staking/stake_initialization.rst

```diff
@@ -246,21 +255,17 @@ bonded a Worker node to your Staker.
 After initiating a stake, the staker must delegate access to a work address through *bonding*.
 There is a 1:1 relationship between the roles: A Staker may have multiple substakes but only ever has one Worker at a time.
 
-.. important:: The Worker cannot be changed for a minimum of 2 periods (48 hours) once bonded.
+.. important:: The Worker cannot be changed for a minimum of 2 periods (14 days) once bonded.
```

Comment:
Wow, we probably should change this parameter to 1 period.

Reply:
Unfortunately, I am not sure that it can be lower than 2 periods. If I remember correctly, 2 periods might the minimum to prevent workers from circumventing slashing... @vzotova ?